test/support/utils.rb

```diff
@@ -51,7 +56,15 @@ def create_pseudo_terminal(boot_options: "-r debug/run")
 
       timeout_sec = (ENV['RUBY_DEBUG_TIMEOUT_SEC'] || 10).to_i
 
-      PTY.spawn("#{RUBY} #{boot_options} #{temp_file_path}") do |read, write, pid|
+      if boot_options == "-r debug/open"
+        r, w, server_pid = PTY.spawn("#{RUBY} #{boot_options} #{temp_file_path}")
+        r.close
+        sleep(0.1)
+        cmd = "exe/rdbg -A"
```

Comment:
should not use this path.

Reply:
https://github.com/ruby/debug/pull/76/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R57